msal4j-brokers/src/main/java/com/microsoft/aad/msal4jbrokers/MsalRuntimeBroker.java

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4jbrokers;
+
+import com.microsoft.aad.msal4j.IAuthenticationResult;
+import com.microsoft.aad.msal4j.IBroker;
+import com.microsoft.aad.msal4j.InteractiveRequestParameters;
+import com.microsoft.aad.msal4j.PublicClientApplication;
+import com.microsoft.aad.msal4j.SilentParameters;
+import com.microsoft.aad.msal4j.UserNamePasswordParameters;
+import com.microsoft.aad.msal4j.MsalClientException;
+import com.microsoft.aad.msal4j.AuthenticationErrorCode;
+import com.microsoft.aad.msal4j.IAccount;
+import com.microsoft.azure.javamsalruntime.Account;
+import com.microsoft.azure.javamsalruntime.AuthParameters;
+import com.microsoft.azure.javamsalruntime.AuthResult;
+import com.microsoft.azure.javamsalruntime.MsalInteropException;
+import com.microsoft.azure.javamsalruntime.MsalRuntimeInterop;
```

Comment:
Is the MsalRuntimeInterop package always available? Will the [line 36](https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/563/files#diff-fb3a656752687c9bd4ce7d08e3912e0fc520cf78a3838e79fff5cce47ce541dbR36) below always succeed?

Reply:
Yes, the msal4jbrokers package has javamsalruntime [as a dependency](https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/dc79a19d0f937e73e1d60c8c2cf23e3f9cdbe4ff/msal4j-brokers/pom.xml#L35), so it and all of its classes (like MsalRuntimeInterop) will always be available.